app/move/views/confirmation.njk

```diff
@@ -0,0 +1,65 @@
+{% extends "layouts/base.njk" %}
+
+{% block customGtagConfig %}
+  gtag('set', {'page_title': 'Move completed'});
+{% endblock %}
+
+{% block pageTitle %}
+  {{ t("moves::confirmation.page_title", { name: fullname | upper }) }}
+{% endblock %}
+
+{% block beforeContent %}
+  {{ super() }}
+
+  {{ govukBackLink({
+    text: t("actions::back_to_dashboard"),
+    classes: "app-print--hide",
+    href: MOVES_URL
+  }) }}
+
+  <a href="javascript:window.print()" class="app-!-float-right app-print--hide govuk-!-margin-top-3">
+    {{ t("actions::print_move_detail") }}
+  </a>
+{% endblock %}
+
+{% block content %}
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+
+      {{ govukPanel({
+        titleText: t('moves::confirmation.panel.heading'),
+        html: t('moves::confirmation.panel.content', {
+          moveReference: moveReference
+        })
+      }) }}
+
+      <p>{{ t("moves::confirmation.detail", {
+          fullname: fullname,
+          location: location,
```

Comment:
Same with location and date I'd say. Just use what is already available via `res.locals.move` and will simplify the controller a lot too.